source/about/certifications-and-compliance.rst

```diff
@@ -55,22 +55,22 @@ Mattermost supports features that ensure data management and data portability.
 
 Data Portability
 ^^^^^^^^^^^^^^^^^
 
-- **Data Import:** Use the `bulk loading tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ to migrate data from an existing messaging system, or for pre-populating a new installation with data. `Review this guide <https://docs.mattermost.com/administration/migrating.html#migrating-from-hipchat-server-and-hipchat-data-center-to-mattermost>`__ which summarizes the different approaches and meets the `Right to Data Portability <https://gdpr-info.eu/art-20-gdpr/>`__ principle.
-- **Data Export:** Use `compliance exports <https://docs.mattermost.com/administration/compliance-export.html>`__ to export conversations from public, private and direct message channels in XML or EML format. Those in Team Edition can export conversations directly from the database, both `in MySQL <https://www.itworld.com/article/2833078/it-management/3-ways-to-import-and-export-a-mysql-database.html>`__ and `in PostgreSQL <https://www.a2hosting.com/kb/developer-corner/postgresql/import-and-export-a-postgresql-database>`__.
+- **Data Import:** Use the `bulk loading tool <https://docs.mattermost.com/onboard/bulk-loading-data.html>`__ to migrate data from an existing messaging system, or for pre-populating a new installation with data. `Review this guide <https://docs.mattermost.com/onboard/migrating-to-mattermost.html#migrating-from-hipchat-server-and-hipchat-data-center-to-mattermost>`__ which summarizes the different approaches and meets the `Right to Data Portability <https://gdpr-info.eu/art-20-gdpr/>`__ principle.
+- **Data Export:** Use `compliance exports <https://docs.mattermost.com/comply/compliance-export.html>`__ to export conversations from public, private and direct message channels in XML or EML format. Those in Team Edition can export conversations directly from the database, both `in MySQL <https://www.infoworld.com/article/3302769/3-ways-to-import-and-export-a-mysql-database.html>`__ and `in PostgreSQL <https://www.a2hosting.com/kb/developer-corner/postgresql/import-and-export-a-postgresql-database>`__.
 
 Accessibility Compliance
 -------------------------
 
 Adherence with accessibility standards is assisted in the following ways:
 
-- **508 Compliance:** For U.S. public sector organizations seeking to confirm 508 compliance, Mattermost publicly shares its `Voluntary Product Accessibility Template (VPAT) online <https://docs.mattermost.com/overview/vpat.html>`__.
+- **508 Compliance:** For U.S. public sector organizations seeking to confirm 508 compliance, Mattermost publicly shares its `Voluntary Product Accessibility Template (VPAT) online <https://docs.mattermost.com/about/vpat.html>`__.
```

Comment:
Note: I made and committed a change to this as we identified a missing page which needed to be re-added.